spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -106,9 +104,20 @@ def enable_b2b_app!
       # Adds users for b2b enabled apps
       def add_b2b_users(user_list = [])
         raise "Cannot add b2b users if b2b is not enabled" unless b2b_app_enabled
-        @b2b_users = user_list.map do |user|
-          B2bUser.from_username(user)
-        end
+        @b2b_users = user_list.map { |user| B2bUser.from_username(user) }
+        self
```

Comment:
Can we change this to `return self`? Explicit is a little easier to read/maintain 😊

Reply:
Sure thing! : )